.github/workflows/codeql.yml

```diff
@@ -31,6 +31,11 @@ on:
 env:
   GO_VERSION: "1.19.x"
 
+permissions:
+  contents: read
+  packages: read
```

Comment:
Is adding these needed? is it ok to leave them as default?

Reply:
I think so; documentation says if they aren't specified, then they will be set to non, so might as be safe

> If you specify the access for any of these scopes, all of those that are not specified are set to none.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions